client/resource_group/controller/model_test.go

```diff
@@ -69,18 +61,6 @@ func TestGetRUTokenBucketSetting(t *testing.T) {
 	if result != expected {
 		t.Errorf("Expected nil but got %v", result)
 	}
-
```

Comment:
Why remove them?

Reply:
stale check, now only `RequestUnitType_RU` type is available cc @nolouch